src/App.scss

```
@@ -1 +1,35 @@
-// styles go here
+html {
+  background-color: black;
+}
+.App {
+  position: absolute;
+  top: 25%;
+  left: 38%;
+  margin-top: 200px;
+  padding-top: 50px;
+  width: 330px;
+  height: 250px;
+  margin: auto;
+  background-color: yellowgreen;
+  border-radius: 20px;
```

Трошки прискіпливості, але загалом, десь в CSS/HTML був [чекліст з порадами](https://codeguide.academy/html-css.html#css-order), і там говорилось про те, щоб окремі групи в CSS блоках розділялися пробілом (в самому кінці). Ну і окремо, в тебе тут `margin: auto` перетирає `margin-top: 200px`, можна його видалити.

```suggestion
  position: absolute;
  top: 25%;
  left: 38%;
  margin-top: 200px;
  padding-top: 50px;
  width: 330px;
  height: 250px;
  margin: auto;
  
  background-color: yellowgreen;
  border-radius: 20px;
```